### DISCUSSION TITLE
More info for OAuth

### DISCUSSION THREAD

**Discussion by Ethan-Hill** [nextauthjs/next-auth#936] [Ideas]
So I've just came to the nuxt-auth module today and was trying out the discord OAuth, i would love to see some more information on the documentation for some of the providers like what is returned after a login, for example discord, it returns name, avatar, email and created at time stamp.

Just an idea but would love to see some more info on the basic setup of a provider instead of just what it needs to auth. 😃

**Comment by LoriKarikari** [comment]
That's a great idea. We could add this to the docs
